3p/integration.js

```diff
@@ -177,6 +177,7 @@ import {mgid} from '#ads/vendors/mgid';
 import {microad} from '#ads/vendors/microad';
 import {miximedia} from '#ads/vendors/miximedia';
 import {mixpo} from '#ads/vendors/mixpo';
+import {momagic} from '#ads/vendors/momagic';
```

Comment:
Please revert the changes to this file. It is not necessary for new vendors.

Reply:
@powerivq 
We have revert the changes to the previous. Please check the same.